spec/requests/incident_with_contracting_oris_and_state_submission_spec.rb

```diff
@@ -130,6 +130,21 @@
       expect(page).to have_content("Submission complete")
     end
 
+    it 'state submission fails if any incidents in the ORI are not complete' do
+      GlobalState.open_submission_window!
+      create_partial_incident :general
+      visit_status :state_submission
+
+      expect(status_nav_count(:draft)).to eq(1)
+      expect(status_nav_count(:approved)).to eq(1)
+
+      click_button 'SUBMISSION'
+      expect(page).to have_content("Your agency has 1 unreviewed incident")
```

Comment:
For completeness, I'd also move the draft to in_review here, and be sure submission still fails.